astacus/coordinator/plugins/clickhouse/manifest.py

```diff
@@ -25,15 +26,26 @@ class AccessEntity(AstacusModel):
     attach_query: bytes
 
     @classmethod
-    def from_plugin_data(cls, data: Mapping[str, Any]) -> "AccessEntity":
-        return AccessEntity(
+    def from_plugin_data(cls, data: Mapping[str, Any]) -> Self:
+        return cls(
```

Comment:
nit. Unrelated change, this should be in a separate commit (same with all similar changes).